libs/core/src/components/sage-checkbox/test/sage-checkbox.e2e.ts

```diff
@@ -0,0 +1,126 @@
+import { newE2EPage } from '@stencil/core/testing';
```

Comment:
All these tests look good to me but can we add some for user flows as well? Can we test when a user clicks on a checkbox and confirm it sets as checked or inverse. Can we confirm invalid states when required show error messages, etc?